fixtures/docstring/tests/bindings/test_docstring.py

```diff
@@ -9,44 +9,45 @@
 from uniffi_docstring import *
 
 # Test function
-assert test.__doc__ == "<docstring-function>"
+assert test.__doc__.strip() == "<docstring-function>"
+assert test_multiline.__doc__.strip() == "<docstring-multiline-function>\n    <second-line>"
 assert test_without_docstring.__doc__ is None
 
 # Test enums
-assert EnumTest.__doc__ == "<docstring-enum>"
+assert EnumTest.__doc__.strip() == "<docstring-enum>"
 
 # Simple enum variants can't be tested, because `__doc__` is not supported for enums
 # assert EnumTest.ONE.__doc__ == "<docstring-enum-variant>"
 # assert EnumTest.TWO.__doc__ == "<docstring-enum-variant-2>"
 
-assert AssociatedEnumTest.__doc__ == "<docstring-associated-enum>"
+assert AssociatedEnumTest.__doc__ .strip() == "<docstring-associated-enum>"
```

Comment:
```suggestion
assert AssociatedEnumTest.__doc__.strip() == "<docstring-associated-enum>"
```

Reply:
Nope, only the missing space infront of the `=` was fixed. There's still an unnecessary space infront of `.strip`

Reply:
doh - I was looking at the wrong `test_docstring.py` 🙃

Reply:
🙈